applications/nrf5340_audio/src/audio/Kconfig.defaults

```diff
@@ -40,6 +40,9 @@ if TRANSPORT_BIS # Depends on TRANSPORT_BIS
 config AUDIO_TEST_TONE
 	default n
 
+config AUDIO_MUTE
```

Comment:
I thought this was changed?

Reply:
Hmm I dont understand the question. I changed from AUDIO_TEST_MUTE to AUDIO_MUTE